src/_cffi_src/utils.py

```diff
@@ -73,6 +73,12 @@ def build_ffi(
     verify_source += '\n#define CRYPTOGRAPHY_PACKAGE_VERSION "{}"'.format(
         about["__version__"]
     )
+    verify_source += r"""
+
+int make_cryptography_openssl_module(void) {
```

Comment:
Let's name this `Cryptography_make_openssl_module()`, which is the prefix we generally use.